org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/XMLLanguageServer.java

```diff
@@ -64,9 +72,29 @@ public XMLLanguageServer() {
 		capabilities.setHoverProvider(true);
 		// capabilities.setExperimental("foldingRangeProvider: true");
 		InitializeResult result = new InitializeResult(capabilities);
+		initializeLoggers(languageClient, params.getInitializationOptions().toString());

```

Comment:
No. params.getInitializationOptions() should be a Map (see https://github.com/eclipse/eclipse.jdt.ls/blob/master/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java#L246)

From that map, you should get the path via options.get("lsp4xml.log")